charts/secrets-store-csi-driver/README.md

```diff
@@ -41,4 +41,5 @@ The following table lists the configurable parameters of the csi-secrets-store-p
 | `livenessProbe.port` | Liveness probe port | `9808` |
 | `livenessProbe.logLevel` | Liveness probe container logging verbosity level | `2` |
 | `rbac.install` | Install default rbac roles and bindings | true |
+| `syncSecrets.enable` | Install rbac roles and bindings required for syncing to Kubernetes native secrets | true |
```

Comment:
we should add a comment for default value here: 
e.g. `true (will update default to false after the next 2 releases) `

We should also add this to the project readme _after_ cutting the next release. tracking issue: https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/268

Reply:
Updated. Let me know if the new wording looks OK. I felt like specifying a specific version might be clearer than a relative number, but I could see referencing #268.